pib/kubefile-schema.yaml

```diff
@@ -0,0 +1,60 @@
+$schema: http://json-schema.org/draft-04/schema#
+
+definitions:
+  cluster:
+    type: object
+    properties:
+      master:
+        type: object
+        properties:
+          size:
+            type: string
+            enum:
+              - t2.nano
+              - t2.micro
+              - t2.small
+              - t2.medium
+              - t2.large
+              - t2.xlarge
+              - t2.2xlarge
+              - m3.medium
+              - m4.large
+              - m4.xlarge
+              - m4.2xlarge
+              - m4.4xlarge
+              - m4.10xlarge
+              - m4.16xlarge
+
+      nodes:
+        type: object
+        properties:
+          size:
+            type: string
+            enum:
+              - t2.nano
+              - t2.micro
+              - t2.small
+              - t2.medium
+              - t2.large
+              - t2.xlarge
+              - t2.2xlarge
+              - m3.medium
+              - m4.large
+              - m4.xlarge
+              - m4.2xlarge
+              - m4.4xlarge
+              - m4.10xlarge
+              - m4.16xlarge
+          nodeCount:
+            type: integer
+
+title: Kubefile
+type: object
+properties:
+  Kubefile-version:
+    type: integer
```

Comment:
Suggest making this string to match `Envfile.yaml`, and so we can do point versions ("2.1").